index.html

```
@@ -2777,5 +2777,87 @@ <h2>Keyboard Shortcuts</h2>
 <script src="src/js/recommender.js"></script>
 <script src="src/js/recommendation-ui.js"></script>
 
-</body>
-</html>
+<button id="scrollTopBtn">↑</button>
+<button id="scrollTopBtn">↑</button>
```

HTMLHint (1.9.2)

[error] The id value [ scrollTopBtn ] must be unique.

(id-unique)